.github/workflows/test.yml

```diff
@@ -176,6 +176,53 @@ jobs:
           name: logs-test-sharedfs-${{ matrix.test }}
           path: /tmp/logs
 
+  test_backend_protected_mode:
+    name: Test backend - protected mode
+    runs-on: ubuntu-latest
+    needs: [build]
+    strategy:
+      matrix:
+        test:
+          - basic status batch anonymous competition unicode rest1 upload1 download refs binary rm make worksheet_search worksheet_tags freeze detach perm search_time groups
```

Comment:
Can we just run `protected_mode` and maybe one other test here? Do we really need to run all tests?

Reply:
All these tests seem to tests different functionalities. I removed a few, but I would like to keep the rest for now for peace of mind.